src/electron/ipc/ipc-channel-names.ts

```diff
@@ -3,3 +3,13 @@
 
 export const IPC_MAIN_WINDOW_INITIALIZED_CHANNEL_NAME = '60dc5587-7f98-4216-92d5-7026875b2ee4';
 export const IPC_MESSAGE_CHANNEL_NAME = '9a436d48-61e9-4c34-849d-3c15ef06f2c9';
+export const IPC_APP_VERSION_CHANNEL_NAME = '48987ad6-02d1-4297-8ab8-57e3a94c83f2';
+export const IPC_BROWSERWINDOW_MAXIMIZE_CHANNEL_NAME = '4edeb851-a52d-4653-acf9-38da8f0048bc';
```

Comment:
not related to this PR. but i am curious why we decided to use guids instead of user friendly ids (like maximize-window etc)